.github/workflows/ci.yaml

```diff
@@ -228,7 +228,7 @@ jobs:
   test_agent_image:
     runs-on: ubuntu-latest
     needs: changed_files
-    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true'}}
+    if: ${{needs.changed_files.outputs.rs_changed == 'true' || needs.changed_files.outputs.ci_changed == 'true' || needs.changed_files.outputs.vscode_changed == 'true'}}
```

Comment:
Why is this necessary?

Reply:
thanks for the review! the `vscode_e2e` job depends on `test_agent_image` and `build_mirrord` and hence they need to run as well to make `vscode_e2e` run

Reply:
I understand that, but why is the vscode job dependent on the agent tests running (as opposed to just building the agent image)?